### PR TITLE
WIP-  Migrate all tests to BK

### PR DESCRIPTION
**Pull request by jaymalasinha**
Signed-off-by: Jaymala Sinha <jsinha@chef.io>

### Description

Migrate the component and oc-pedant tests to BK too.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [x] New functionality includes tests
- [ ] All tests pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: cookbooks/chef-server-deploy/recipes/omnibus_standalone.rb [hunk 1]
```diff
@@ -141,7 +141,7 @@
 
 file '/var/opt/opscode/nginx/etc/addon.d/99-supermarket-credentials_external.conf' do
   content(lazy do
-    <<-EOF
+<<-EOF
```
Comment (markan) [inline cookbooks/chef-server-deploy/recipes/omnibus_standalone.rb]: Looks like the cookstyle test dislikes this change:

    recipes/omnibus_standalone.rb:144:1: C: Layout/IndentationWidth: Use 2 (not -2) spaces for indentation.

File: .expeditor/verify.pipeline.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ steps:
   expeditor:
     executor:
       docker:
+        image: "chefes/lita-worker"
```
Comment (tduffield) [inline .expeditor/verify.pipeline.yml]: We should never use this image in Buildkite. We should always use `chefes/buildkite`.

File: .expeditor/verify.pipeline.yml [hunk 3]
```diff
@@ -84,8 +79,8 @@ steps:
         privileged: true
         image: "chefes/a1-buildkite"
         environment:
-            - OCTOKIT_ACCESS_TOKEN
-            - COMPONENT=src/oc_erchef
+          - OCTOKIT_ACCESS_TOKEN
```
Comment (tduffield) [inline .expeditor/verify.pipeline.yml]: Do we really need `OCTOKIT_ACCESS_TOKEN` in all these tests? to my knowledge that env var is specific to the octokit ruby library.

**Comment by jaymalasinha** [comment]
Closing in favor of #1640 Incorporated the review feedback in the cleaner branch

**Pull request closed without merging**